drop-in/src/main/java/com/adyen/checkout/dropin/ui/paymentmethods/PaymentMethodAdapter.kt

```diff
@@ -96,40 +77,40 @@ class PaymentMethodAdapter(
         holder.detail.visibility = View.VISIBLE
     }
 
+    private fun bindGenericStored(holder: StoredPaymentMethodVH, genericStoredModel: GenericStoredModel) {
+        holder.text.text = genericStoredModel.name
+        holder.detail.visibility = View.GONE
+        imageLoader.load(genericStoredModel.imageId, holder.logo)
+    }
+
     private fun bindPaymentMethod(holder: PaymentMethodVH, position: Int) {
         val paymentMethod = getPaymentMethodAt(position)
 
         holder.text.text = paymentMethod.name
         holder.detail.visibility = View.GONE
 
-        val txVariant =
-            if (paymentMethod.type == PaymentMethodTypes.SCHEME) CARD_LOGO_TYPE
-            else paymentMethod.type
-
-        imageLoader.load(txVariant, holder.logo)
+        holder.logo.setRoundingDisabled(!paymentMethod.drawIconBorder)
```

Comment:
Don't you also need to invert this logic after changed the boolean to true? Then `setRoundingDisabled` would be just `setRounding`?

Reply:
Haha yeah it's a weird one, cause the lack of rounding is the exception here. But it doesn't make sense to change it in one place but not the other, will fix 👍